lib/ansible/modules/cloud/ovirt/ovirt_storage_domains.py

```diff
@@ -551,7 +570,11 @@ def main():
                 storage_domain=storage_domains_service.service(ret['id']).get()
             )
             ret['changed'] = storage_domains_module.changed
-
+        elif state == 'update_ovf_store':
+            ret = storage_domains_module.action(
+                action='update_ovf_store'
+            )
```

Comment:
Can we wait here somehow for the operation to end?

Reply:
Since the storage domain is not locked, there is no indication which we can use to notify when the operation was ended besides the audit log
shouldn't adding 'wait' to the task support it ?

Reply:
Well, by default if you pass `wait=true`, we use some condtion to happen, but you need to write that condition. We already rely in audit log messages, because unfurtunatelly sometimes we must. Take a look here, how to implement it using audit log:

 https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/ovirt/ovirt_hosts.py#L474

Reply:
Unfortunately, the update of the OVF_STORE does not audit log in case there was no update. I think that we should probably add this gap in the engine first and then fix it in ansible, or alternatively can't we use REST to use some kind of sync=true?

BTW That is great to know, I was looking how to do this kind of 'wait' because that is what I was missing to make attach/detach/activate to be done simultaneously for several storage domains. unless you have another way to do that

Reply:
Sync/Async=true/false, doesn't work how you expect unfortunatelly. So the only way to wait for result is to poll the status of the object.

So please add comment that `wait=true` don't work for `update_ovf_store` .

 - "I(update_ovf_store) is supported since version 2.5, currently if C(wait) is (true), we don't wait for update."

Reply:
done

Reply:
done